spec/models/services/tumblr_spec.rb

```diff
@@ -11,14 +11,16 @@
 
   describe '#post' do
     it 'posts a status message to tumblr and saves the returned ids' do
-      response = double(body: '{"response": {"user": {"blogs": [{"url": "http://foo.tumblr.com"}]}}}')
+      response = double(body: '{"response": {"user": {"blogs":
+       [{"primary": false, "url": "http://foo.tumblr.com"},
+        {"primary": true, "url": "http://bar.tumblr.com"}]}}}')
       expect_any_instance_of(OAuth::AccessToken).to receive(:get)
       .with("/v2/user/info")
       .and_return(response)
 
       response = double(code: "201", body: '{"response": {"id": "bla"}}')
       expect_any_instance_of(OAuth::AccessToken).to receive(:post)
-      .with("/v2/blog/foo.tumblr.com/post", @service.build_tumblr_post(@post, ''))
+      .with("/v2/blog/bar.tumblr.com/post", @service.build_tumblr_post(@post, ''))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.